include/fmt/format.h

```diff
@@ -2992,6 +2992,7 @@ FMT_CONSTEXPR const Char* parse_replacement_field(const Char* begin,
 template <bool IS_CONSTEXPR, typename Char, typename Handler>
 FMT_CONSTEXPR_DECL FMT_INLINE void parse_format_string(
     basic_string_view<Char> format_str, Handler&& handler) {
+  using detail::find;
```

Comment:
Could you add a short comment saying that this is a workaround for an MSVC bug?

Reply:
Sure.